src/main/kotlin/Decorator/littlekai/ingredients/Chicken.kt

```diff
@@ -4,6 +4,6 @@ import Decorator.littlekai.base.IngredientDecorator
 import Decorator.littlekai.base.Noodles
 
 
-class Chicken(noodles: Noodles) : IngredientDecorator(noodles) {
-  override val COST: Double = 3.50
+class Chicken(noodles: Noodles, val COST: Double = 3.50): IngredientDecorator(noodles) {
```

Comment:
But you are placing the declaration in the constructor. You are giving the opportunity to set another `COST` when making a new instance.